[Edit] HTML Add Media (HTML Attribute) (#3186)
Codecademy/docs: content/html/concepts/attributes/attributes.md

Author: @SSwiniarski please review my work, I have gone thoroughly over your comments to make changes, to this file.
Please note the image in the gif showcasing the img attribute is taken by me if you want I can change it to showcase a simple shape, if it's okay with you feel free to use my image, I'd be proud to.

  - 'paths/front-end-engineer-career-path'
---

HTML attributes are data that can be added to the opening tag of an HTML element. These attributes modify the default behavior of the element or provide additional information about it. 
Attributes consist of a name and are usually set equal to a value. Values are enclosed in double or single quotation marks. 

## Syntax 

The syntax for adding attributes to an HTML element is as follows: 

 ```html 
 <tagname attribute="value"></tagname> 
 ```

> **Note:** Attribute names are not case-sensitive, but it is good practice to write HTML markup in lowercase.
 
## Examples 

 ### Image Attributes

> Using an  `<img>` element, which is commonly used to display images, reveals several **attributes**, such as `src`, `width`, and `height`. These attributes exert significant influence over the manner in which the image appears on the web page. The dimensions of the element in particular, once specified, enable precise control over the end result.

Reviewer: [proofreading] Always hyperlink elements that were written about. I highly recommend checking out the [Codecademy Doc's website](https://www.codecademy.com/resources/docs) to find them.  
```suggestion
Using an  [`<img>`](https://www.codecademy.com/resources/docs/html/elements/img) element, which is commonly used to display images, reveals several **attributes**, such as `src`, `width`, and `height`. These attributes exert significant influence over the manner in which the image appears on the web page. The dimensions of the particular element, once specified, enable precise control over the end result.
```

```html
<img src="source_of_image.png" width="500" height="600" />
```

![GIF showcasing effect of height attribute on an img element](https://raw.githubusercontent.com/Codecademy/docs/main/media/changing-height-attribute-of-html-element.gif)

 ###  `disabled` Attribute 

The `disabled` attribute is applied to form elements like buttons and inputs to make them non-interactable:

```html
<button disabled>Submit</button>
```

> ![Disable attribute on a <button> element](https://raw.githubusercontent.com/Codecademy/docs/main/media/media/button-disabled-attribute.gif)
> 

Reviewer: [nitpick] While showing the output is helpful, adding introductory text that indicates this would clarify the example more. 
```suggestion
This is the output of the above code snippet: 
![Disable attribute on a <button> element](https://raw.githubusercontent.com/Codecademy/docs/main/media/media/button-disabled-attribute.gif)

```

  - 'paths/front-end-engineer-career-path'
---

> HTML attributes are data that can be added to the opening tag of an HTML element. These attributes modify the default behavior of the element or provide additional information about it. 

Reviewer 2: ```suggestion
HTML **attributes** are data that can be added to the opening tag of an HTML element. These attributes modify the default behavior of the element or provide additional information about it. 

```

Attributes consist of a name and are usually set equal to a value. Values are enclosed in double or single quotation marks. 

## Syntax 

The syntax for adding attributes to an HTML element is as follows: 

 ```html 
 <tagname attribute="value"></tagname> 
 ```

> **Note:** Attribute names are not case-sensitive, but it is good practice to write HTML markup in lowercase.
 
## Examples 

 ### Image Attributes

> Using an  [`<img>`](https://www.codecademy.com/resources/docs/html/elements/img) element, which is commonly used to display images, reveals several **attributes**, such as `src`, `width`, and `height`. These attributes exert significant influence over the manner in which the image appears on the web page. The dimensions of the particular element, once specified, enable precise control over the end result.

Reviewer 2: ```suggestion
Using an  [`<img>`](https://www.codecademy.com/resources/docs/html/elements/img) element, which is commonly used to display images, reveals several attributes, such as `src`, `width`, and `height`. These attributes exert significant influence over the manner in which the image appears on the web page. The dimensions of the particular element, once specified, enable precise control over the end result.
```

```html
<img src="source_of_image.png" width="500" height="600" />
```

> ![GIF showcasing effect of height attribute on an img element](https://raw.githubusercontent.com/Codecademy/docs/main/media/changing-height-attribute-of-html-element.gif)

Reviewer 2: ```suggestion
![GIF showcasing effect of height attribute on an img element](https://raw.githubusercontent.com/Codecademy/docs/main/media/Changing-height-attribute-of-html-element.gif)
```

Author: Hi  @caupolicandiaz , I have committed the changes from your review, Unfortunately, something is wrong with my 
local repo of Codecademy docs, and it's missing the documents that I have committed not 100% sure what happened therefore I can't or don't seem to understand how to fix the capital letter in the directory expect to start from scratch in new repo. Could you assist me in it or let the name be like so?

Reviewer 2: The file doesn't need to be changed. The link did not match the existing case, so the change put them in alignment

 ###  `disabled` Attribute 

> The `disabled` attribute is applied to form elements like buttons and inputs to make them non-interactable:

Reviewer 2: ```suggestion
The `disabled` attribute is applied to form elements like buttons and inputs to make them nonfunctional:
```

```html
<button disabled>Submit</button>
```

This is the output of the above code snippet: 

> ![Disable attribute on a <button> element](https://raw.githubusercontent.com/Codecademy/docs/main/media/media/button-disabled-attribute.gif)

Reviewer 2: ```suggestion
![Disable attribute on a <button> element](https://raw.githubusercontent.com/Codecademy/docs/main/media/button-disabled-attribute.gif)
```

  - 'paths/front-end-engineer-career-path'
---

> 
> # HTML Attributes 

Reviewer 3: ```suggestion
```
We do not use opening h1 headers in Docs, the Title in the metadata provides that.

> HTML attributes are essential elements that can be added to the opening tag of an HTML element. These attributes modify the default behavior of the element or provide additional information about it. 
> 
> Attributes consist of a name and need to be set off equal to a value. Values are enclosed in double or single quotation marks. 

Reviewer 3: ```suggestion
HTML attributes are data that can be added to the opening tag of an HTML element. These attributes modify the default behavior of the element or provide additional information about it. 

Attributes consist of a name and are usually set equal to a value. Values are enclosed in double or single quotation marks. 
```

> ## Syntax 
> The syntax for adding attributes to an HTML element is as follows: 
>  ```html 
>  <tagname attribute="value"></tagname> 
>  ```

Reviewer 3: ```suggestion

## Syntax 

The syntax for adding attributes to an HTML element is as follows: 

  ```pseudo 
 <tagname attribute="value"></tagname> 
 ```
```

 
## Examples 

>  1. **Image Attributes**

Reviewer 3: ```suggestion
### Image Attributes
```

> Consider an `<img>` element, commonly used to display images. It can have several **attributes**, such as `src`, `width`, and `height`, which significantly influence how the image is presented on a web page. As we have declared its dimentions to our element. Hence we control the deserible output of the illustration inside of our HTML file.
> ```html
> <img src="source_of_image.png" width="500" height="600" />

Reviewer 3: ````suggestion
Consider an `<img>` element, commonly used to display images. It can have several attributes, such as `src`, `width`, and `height`, which significantly influence how the image is presented on a web page. 

```html
<img src="source_of_image.png" width="500" height="600" />
````
We don't use first- or second-person pronouns in docs.

```

> ![GIF showcasing effect of height attribute on an img element](../../../../media/Changing-height-attribute-of-html-element.gif)

Reviewer 3: Please re-read the [style guide on image hosting](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md#file-hosting). The link to use needs to be `https://raw.githubusercontent.com/Codecademy/docs/main/media/` followed by the image name.

>  2. ### `href` Attribute (Hyperlink)

Reviewer 3: `href` is not a good example because it's too hard to illustrate with an image. I'd remove this section.

The `href` attribute is used in anchor (`<a>`) tags to specify the URL to which a link leads. Here's an example:
```html
<a href="https://www.example.com" >Visit Example</a>
```
![Visiting a link via href attribute on <a> ](../../../../media/href-attribute.png)

> 3. ###  `disabled` Attribute (Form Elements)

Reviewer 3: ```suggestion
###  `disabled` Attribute
```

> The `disabled` attribute is applied to form elements like buttons and inputs to make them non-interactable:
> ```html
> <button disabled>Submit</button>
> ```

Reviewer 3: ````suggestion

The `disabled` attribute is applied to form elements like buttons and inputs to make them non-interactable:

```html
<button disabled>Submit</button>
```
````

> ![Disabling attribute on a <button> element](../../../../media/button-disabled-attribute.png)

Reviewer 3: See my prior comment on the form of image links.

3. ###  `disabled` Attribute (Form Elements)

These declare its source, width, and height.
The `disabled` attribute is applied to form elements like buttons and inputs to make them non-interactable:
```html
<button disabled>Submit</button>
```
![Disabling attribute on a <button> element](../../../../media/button-disabled-attribute.png)

> **Note:** Attribute names are not case-sensitive, but it is good practice to write HTML markup in lowercase.

Reviewer 3: Restore this note and add it back in the "syntax" section.

> ## Conclusion
> These are just a few examples of HTML attributes. HTML provides a wide range of attributes that enhance the functionality and styling of web pages, making it a powerful tool for web development.

Reviewer 3: ```suggestion
Below are examples of commonly used attributes.
```
We generally don't have a conclusion section. Instead, yo should introduce the list of terms shown at the end of the entry.
